lib/generators/rafinesque/install_generator.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails/generators'
+
+module Rafinesque
+  module Generators
+    class InstallGenerator < Rails::Generators::Base
+
+      source_root File.expand_path('../../../rafinesque', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.